fem/lininteg.hpp

```diff
@@ -456,6 +456,12 @@ class VectorFEBoundaryFluxLFIntegrator : public LinearFormIntegrator
                                        Vector &elvect);
 
    using LinearFormIntegrator::AssembleRHSElementVect;
+
+   virtual bool SupportsDevice() { return true; }
```

Comment:
`LinearFormIntegrator::SupportsDevice` should really be const. Should I make that change here? cc: @camierjs

Reply:
Done in 9732e4dc6f46a1d194567fb022db0564fc635cb7